msgpack-core/src/main/java/org/msgpack/core/MessagePack.java

```diff
@@ -518,7 +518,7 @@ public boolean isStr8FormatSupport()
 
         private int stringSizeLimit = Integer.MAX_VALUE;
 
-        private int bufferSize = 8192;
+        private int bufferSize = 100 * 1024 * 1024;
```

Comment:
Don't change the default buffer size. msgpack-java is frequently used even for the small size of data.

Reply:
Especially, creating a large buffer is not free as Java will use CPU resources for filling zeros to the array contents.